main.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/bitrise-io/go-utils/log"
 	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-io/go-utils/ziputil"
+	xcarchive "github.com/bitrise-io/go-xcode/xcarchive"
```

Comment:
no need for named import, as it is equivalent to the default import name of the package

Reply:
Fixed.